src/backend/xilinx/memory_axi.rs

```diff
@@ -293,18 +305,22 @@ fn module_mode_fsm(module: &mut v::Module) -> fsm::LinearFsm {
 }
 
 fn bram_logic(
+    index: u64,
```

Comment:
I'd prefer this to be the name instead of an instance. That way, it's the responsibility of the caller to ensure the name is unique